oauthlib/oauth2/rfc6749/request_validator.py

```diff
@@ -378,104 +352,6 @@ def save_bearer_token(self, token, request, *args, **kwargs):
         """
         raise NotImplementedError('Subclasses must implement this method.')
 
-    def get_jwt_bearer_token(self, token, token_handler, request):
```

Comment:
Could this one not be used in an OAuth 2.0 flow?

Reply:
Indeed, however I'd suggest to change it again the other way around if ever it's added into an OAuth2 grant (currently it is only called from openid grant).